test/libyul/yulSyntaxTests/invalid_tuple_assignment.yul

```diff
@@ -2,4 +2,4 @@
 	let x, y := 1
 }
 // ----
-// DeclarationError 3812: (3-16): Variable count mismatch: 2 variables and 1 values.
+// DeclarationError 3812: (3-16): Variable count mismatch for "x, y": 2 variables and 1 values.
```

Comment:
Maybe this could be `Variable count mismatch for declaration of "x, y"` / "...assignment to "x, y"

Reply:
Fixed these two cases.